op-deployer/pkg/cli/app.go

```diff
@@ -26,6 +26,25 @@ func NewApp(versionWithMeta string) *cli.App {
 		}
 		return nil
 	}
+
+	bootstrapImplementationsFlags := append(bootstrap.ImplementationsFlags, deployer.AutoVerifyFlag, deployer.VerifierFlag, deployer.VerifierUrlFlag, deployer.VerifierAPIKeyFlag)
+	bootstrapSuperchainFlags := append(bootstrap.SuperchainFlags, deployer.AutoVerifyFlag, deployer.VerifierFlag, deployer.VerifierUrlFlag, deployer.VerifierAPIKeyFlag)
+
+	bootstrapCommandsWithVerify := []*cli.Command{
+		{
+			Name:   "implementations",
+			Usage:  "Bootstraps implementations.",
+			Flags:  cliapp.ProtectFlags(bootstrapImplementationsFlags),
+			Action: bootstrap.ImplementationsCLI,
+		},
+		{
+			Name:   "superchain",
+			Usage:  "Bootstrap the Superchain configuration",
+			Flags:  cliapp.ProtectFlags(bootstrapSuperchainFlags),
+			Action: bootstrap.SuperchainCLI,
+		},
+	}
+
```

Comment:
Can we get rid of this code and just add the verification flags directly within `bootstrap/flags.go`? That would remove some indirection and make the code easier to follow IMO